mon-pix/translations/en.json

```
@@ -17,7 +17,11 @@
     },
     "french-republic": "French Republic",
     "level": "level",
-    "loading": "Loading",
+    "loading": {
+      "default": "Loading",
+      "test": "Your test is loading",
+      "results": "Preparing your results"
```

remettre les clés par ordre alphabétique:
```suggestion
"results": "Preparing your results",
"test": "Your test is loading"
```

---

Done